src/scripts/Player.ts

```diff
@@ -440,9 +442,13 @@ class Player {
         // TODO add money multipliers
         let oakItemBonus = OakItemRunner.isActive("Amulet Coin") ? (1 + OakItemRunner.calculateBonus("Amulet Coin") / 100) : 1;
         let moneytogain = Math.floor(money * oakItemBonus * (1 + AchievementHandler.achievementBonus()))
+        if(this.effectEngine['Lucky_incense']){
+            moneytogain = Math.floor(moneytogain * 1.5);
```

Comment:
My memory might not be correct but isn't there a battleItem created somewhere with this multiplier?

Reply:
not sure what you mean, the only enums i see for incense is the price and new declared string i built